internal/cli/kraft/cloud/utils/print.go

```diff
@@ -600,6 +600,70 @@ func PrintLimits(ctx context.Context, format string, resp *kcclient.ServiceRespo
 	return table.Render(iostreams.G(ctx).Out)
 }
 
+func PrintFeatures(ctx context.Context, format string, resp *kcclient.ServiceResponse[kcusers.QuotasResponseItem]) error {
```

Comment:
Maybe `PrintQuotaFeatures` to provide better context?

Reply:
same for `PrintLimits` I guess then